app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt

```diff
@@ -1174,8 +1178,12 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener {
             null
         }
 
-        changeStatusBarState(isEmulatorSettings())
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && !isAppOptimised && !isTrueTvSettings()
```

Comment:
Another annoying popup when installing the app, this is not what we want. This should only show up when actually trying to download stuff.